Source/WarpX.H

```diff
@@ -1137,16 +1148,33 @@ private:
     }
 #endif
 
+public:
     void InitEB ();
-
     void ComputeEdgeLengths ();
     void ComputeFaceAreas ();
     void ScaleEdges ();
     void ScaleAreas ();
+    void MarkCells();
+    amrex::Array1D<int, 0, 2> CountExtFaces();
```

Comment:
Please add Doxygen docs here in the declaration of these methods (Doxygen will not find them if you put them in the `.cpp` file)
https://warpx.readthedocs.io/en/latest/developers/documentation.html